.github/workflows/build.yml

```
@@ -414,6 +414,35 @@ jobs:
       - name: Build and test
         run: ./Android/android.py ci --fast-ci ${{ matrix.arch }}-linux-android
 
+  build-ios:
+    name: iOS (${{ matrix.arch }})
+    needs: build-context
+    if: needs.build-context.outputs.run-tests == 'true'
+    timeout-minutes: 60
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - arch: aarch64
+            runs-on: macos-15
+    runs-on: ${{ matrix.runs-on }}
```

And we can use this directly too:
```suggestion
    runs-on: macos-15
```